.tugboat/config.yml

```diff
@@ -1,20 +1,41 @@
 services:
-    apache:
-        image: tugboatqa/httpd
-        commands:
-            init:
-                - curl -Ls https://github.com/gohugoio/hugo/releases/download/v0.58.3/hugo_0.58.3_Linux-64bit.tar.gz | tar -C /usr/local/bin -zxf - hugo
-                - sed -i '/mod_rewrite/s/#//g' /usr/local/apache2/conf/httpd.conf
-                - sed -i 's/AllowOverride None/AllowOverride All/g' /usr/local/apache2/conf/httpd.conf
-            build:
-                - hugo
-                - ln -snf "${TUGBOAT_ROOT}/public" "${DOCROOT}"
+  apache:
+    image: tugboatqa/httpd
+    aliases:
+      - foo
+      - bar
+      - baz.example.com
```

Comment:
did you mean for these to get added?

```suggestion
```

Reply:
Oh, I think I understand what you're doing. You want these to be there so that the environment variable will be there in the documentation. I'm not crazy about this, though, because these URLs will show up in the comment that is posted to any PR, which will create confusion for contributors. Do you have other ideas on how to achieve this?

Reply:
Makes sense.  I'll comment them out and note that in the documentation for updating the variables.